dlp/snippets/deid/deid_free_text_with_fpe_using_surrogate_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package deid
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestDeidentifyFreeTextWithFPEUsingSurrogate(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+
+	inputStr := "My phone number is 1234567890"
+	infoType := "PHONE_NUMBER"
+	surrogateType := "PHONE_TOKEN"
+	unWrappedKey, err := getUnwrappedKey(t)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if err := deidentifyFreeTextWithFPEUsingSurrogate(&buf, tc.ProjectID, inputStr, infoType, surrogateType, unWrappedKey); err != nil {
+		t.Fatal(err)
+	}
+
+	got := buf.String()
+	if want := "output: My phone number is "; !strings.Contains(got, want) {
+		t.Errorf("reidentifyFreeTextWithFPEUsingSurrogate got %q, want %q", got, want)
```

Comment:
This error doesn't match the test name. A couple more errors don't match the method name, plese fix all.

Reply:
updated it with method name instead of test name.